package.json

```diff
@@ -0,0 +1,107 @@
+{
+  "name": "@logdna/exclusive-lock",
+  "description": "Create atomic locks that can be used in a distributed environment",
+  "version": "0.0.0",
+  "main": "index.js",
+  "files": [
+    "lib/**",
+    "index.js",
+    "README.md",
+    "LICENSE"
+  ],
+  "scripts": {
+    "tap": "tap",
+    "lint": "eslint ./",
+    "lint:fix": "npm run lint -- --fix",
+    "local": "env $(cat env/local.env) ",
+    "commitlint": "commitlint --from=origin/main --to=HEAD",
+    "release": "semantic-release",
+    "release:dry": "semantic-release --dry-run --no-ci --branches ${BRANCH_NAME:-main}",
+    "pretest": "npm run lint && npm run commitlint",
+    "test": "tools/test-ci.sh",
+    "test:ci": "docker-compose -f compose/base.yml -f compose/test.yml up --always-recreate-deps --force-recreate --remove-orphans --build --abort-on-container-exit",
+    "start": "docker-compose -f compose/base.yml -f compose/dev.yml up",
+    "stop": "docker-compose -f compose/base.yml -f compose/dev.yml down --remove-orphans"
+  },
+  "author": "Darin Spivey <darin.spivey@mezmo.com>",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/logdna/exclusive-lock-node"
+  },
+  "homepage": "https://github.com/logdna/exclusive-lock-node",
+  "bugs": {
+    "url": "https://github.com/logdna/exclusive-lock-node/issues"
+  },
+  "license": "SEE LICENSE IN LICENSE",
+  "publishConfig": {
+    "access": "public"
+  },
+  "commitlint": {
+    "extends": [
+      "@logdna/commitlint-config-mezmo"
+    ]
+  },
+  "private": false,
+  "release": {
+    "extends": "semantic-release-config-logdna",
+    "branches": [
+      "main"
+    ]
+  },
+  "eslintConfig": {
+    "root": true,
+    "ignorePatterns": [
+      "node_modules/",
+      "coverage/"
+    ],
+    "extends": "logdna",
+    "parserOptions": {
+      "ecmaVersion": 13
+    }
+  },
+  "engineStrict": true,
+  "engines": {
+    "node": ">=14"
+  },
+  "tap": {
+    "100": true,
+    "ts": false,
+    "jsx": false,
+    "browser": false,
+    "check-coverage": true,
+    "output-file": ".tap-output",
+    "files": [
+      "test/unit",
+      "test/integration"
+    ],
+    "coverage-report": [
+      "text",
+      "text-summary",
+      "json",
+      "json-summary",
+      "html"
+    ],
+    "nyc-arg": [
+      "--all",
+      "--exclude=test/",
+      "--exclude=coverage/"
+    ]
+  },
+  "devDependencies": {
+    "@logdna/commitlint-config-mezmo": "^1.0.1",
```

Comment:
ya, I'm not sure it make sense to be putting mezmo packages in the logdna name space. 
I think we should kill this problem before it starts. 

The answerbook/logdna problem is confusing enough. can we just publish it with out a name space, `mezmo` is in the package name. we do that for the lint packages.

Reply:
Creating a mesbo namespace failed. Wasn't sure what else to do. All of our public packages are published under that namespace.

Sounds like a bigger discussion

Reply:
The namespace for the package has been removed